### PR TITLE
[RFC] test: Migrate legacy test 88.

### PR DESCRIPTION
**Pull request by lucc**
There is a minor oddity going on in this test:  With the first commit I tried to translate the test as literally as possible but I decided that the `:normal` commands could be replaced by `feed()` calls.  But in four places this would actually have changed the output so I left the commands.  But when I modernize the test and use `eq()` and `eval()` to check the output the problem doesn't arise.

### CODE REVIEW

**Comment by lucc** [comment]
ref for better overview: #1286

**Comment by justinmk** [comment]
@lucc Really appreciate your help with the tests. It may be a few days before I can start reviewing them again.

**Comment by lucc** [comment]
@justinmk I understand that migrating legacy tests has been prosponed to the 0.2 milestone. I will have a look trough the 0.1 milestone and see what I can do because my original motivation when migrating tests was the note in the readme that this is top priory atm but I didn't see it happen.

**Comment by lucc** [comment]
I think the build failure of "default" is unrelated (and travis and appveyor did pass).

File: test/functional/legacy/088_conceal_tabs_spec.lua [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    expect_pos(5, 25)
+    feed('$')
+    expect_pos(5, 28)
+    feed('0j')
+    expect_pos(6, 8)
+    feed('ft')
+    expect_pos(6, 25)
+    feed('$')
+    expect_pos(6, 28)
+    feed('j0j')
```
Comment (justinmk) [inline test/functional/legacy/088_conceal_tabs_spec.lua]: this is `0j0j` in the original test

**Comment by justinmk** [comment]
LGTM, just have the one question above

**Comment by lucc** [comment]
I fixed your comment (although the first `0` would have no effect in the end if we do `0` one line below, that's why the test didn't fail).

**Pull request merged**